server/models/industryData.js

```diff
@@ -0,0 +1,61 @@
+const mongoose = require('mongoose');
+
+const jobSchema = new mongoose.Schema({
+  employer: {
+    type: String,
+    required: true,
+  },
+  title: {
+    type: String,
+    required: true,
+  },
+  isInternship: {
+    type: Boolean,
+    required: true,
+  },
+  isFullTime: {
+    type: Boolean,
+    required: true,
+  },
+  description: {
+    type: String,
+    reuquired: true,
+  },
+  location: {
+    type: String,
+    required: true,
+  },
+  reqYearsExp: {
+    type: Number,
+    required: true,
+  },
+  tags: [String],
+  timeCommitment: String,
+  pay: String,
+  deadline: Date,
+  startDate: Date,
+  endDate: Date,
+  datePosted: {
+    type: Date,
+    default: Date.now,
+  },
+});
+
+const industryDataSchema = new mongoose.Schema({
+  jobs: {
+    active: {
+      type: [jobSchema],
+      default: [],
+    },
+    draft: {
+      type: [jobSchema],
+      default: [],
+    },
+    archived: {
+      type: [jobSchema],
+      default: [],
+    },
+  },
+});
+
```

Comment:
You can write the schema how ever you'd like since it is a separate part of the project, just ensure that the server documentation is updated in the models section so that your ideas for creating the schema are clearly recorded as this does not follow the same format as the other faculty schemas